client/payments/data/actions.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import { apiFetch } from '@wordpress/data-controls';
+
+/**
+ * Internal dependencies
+ */
+import { ACTION_TYPES as TYPES } from './action-types';
```

Comment:
What was the reason for aliasing this import?

Reply:
🤦 no particular reason actually. I blindly copied and used it from other existing TS scripts. I will fix it.